more_itertools/more.py

```diff
@@ -3035,34 +3035,49 @@ def set_partitions_helper(L, k):
         yield from set_partitions_helper(L, k)
 
 
-def time_limited(limit_seconds, iterable):
+class time_limited:
     """
     Yield items from *iterable* until *limit_seconds* have passed.
+    If the iterable is not exhausted in the time limit, the ``timed_out``
```

Comment:
I think:

`If the time limit is reached before the iterable is exhausted, the ``timed_out`` attribute will be ``True``
`

would be clearer.